compat/src/suspense-list.js

```diff
@@ -0,0 +1,72 @@
+import { Component, options } from 'preact';
+import { Suspense } from './suspense';
+
+// Hook for Suspense boundaries to ask for any extra work before rendering suspended children.
+options.__onSuspensionComplete = (vnode, cb) => {
+	if (vnode._parent._component && vnode._parent._component.__modifySuspense) {
+		vnode._parent._component.__modifySuspense(vnode, cb);
+	} else {
+		cb();
+	}
+};
+
+// having custom inheritance instead of a class here saves a lot of bytes
+export function SuspenseList(props) {
+	// Todo(prateekbh): use a better name before sending the PR.
+	this._thrillers = [];
+}
+
+// Things we do here to save some bytes but are not proper JS inheritance:
+// - call `new Component()` as the prototype
+// - do not set `Suspense.prototype.constructor` to `Suspense`
+SuspenseList.prototype = new Component();
+
+SuspenseList.prototype.__modifySuspense = function(vnode, cb) {
+	switch (this.props.revealOrder) {
```

Comment:
Does replacing this switch with an `if` statement using the nifty `props.revealOrder > 't'` trick save us some bytes? 

Perhaps we could get away with the default/`else` case being `forwards`. Using `SuspenseList` without an explicit revealOrder seems like bug ([React warns on it](https://github.com/facebook/react/blob/0f3838a01b0fda0ac5fd054c6be13166697a113c/packages/react-reconciler/src/ReactFiberBeginWork.js#L2197)) and I'd guess the most common case would be `forwards` so perhaps we can save bytes and just use `forwards` as the default case?

Reply:
I would go for

```
if (revealOrder[0] === 'f') ....
```

Reply:
Wondering if we should go for @jviide's suggestions though:
```
> q = x => (x < "f") + (x < "t")
> q("forwards")
1
> q("backwards")
2
> q("together")
0
```
🤔  such a great small snippet

Reply:
we'll always need an if because the along with these values it can be non existing as well

so in my head it'll end up looking like
`if(props.revealOrder){}else{switch q ....}`

cuz q(null)===0.

Thus to keep the balance between byte savings and readability, i'd vote for `if (revealOrder[0] === 'f') ....`

Reply:
done